### PR TITLE
Adding imagery of Rijeka, Croatia

### PR DESCRIPTION
**Pull request by Janjko**
Added 7 files for 7 years of aerial imagery (2000., 2002., 2005., 2008., 2011., 2012. and 2020.). The boundaries are the same in all `.geojson` files, although the imagery doesn't cover exactly the same area.

Also, an icon of City of Rijeka coat of arms .

### CODE REVIEW

**Comment by rbuffat** [comment]
@Janjko You can use https://mybinder.org/v2/gh/rbuffat/eli-helper/master?urlpath=apps%2F/create_wms_template_appmode.ipynb to find out the available_projections parameter.

**Comment by Janjko** [comment]
> 
> 
> @Janjko You can use https://mybinder.org/v2/gh/rbuffat/eli-helper/master?urlpath=apps%2F/create_wms_template_appmode.ipynb to find out the available_projections parameter.

Thanks. I already found them in GetCapabilities, but there were more than 5000 available projections. So I just took the ones we used in other wms services in Croatia.

The strict check failed because of the icon which is not there because it's in this PR. The second error is `ERROR:root:Layers 'CKAN:Ortofoto_2012' not advertised by WMS GetCapabilities request.` with which I'm not sure what's the next step.

**Comment by Janjko** [comment]
[Here is the GetCapabilities xml](http://gis01.rijeka.hr/geoserver/CKAN/wms?service=WMS&version=1.1.0&request=GetCapabilities)

**Comment by rbuffat** [comment]
The server seems to list the following layers:

Layers:

> 0: GRANICA GRADA (Granica grada Rijeke)
> 1: MJESNI_ODBORI (Mjesni odbori)
> 2: MORSKA_GRANICA (Morska granica grada Rijeke)
> 3: Ortofoto_2000 (Ortofoto 2000)
> 4: Ortofoto_2002 (Ortofoto 2002)
> 5: Ortofoto_2005 (Ortofoto 2005)
> 6: Ortofoto_2008 (Ortofoto 2008)
> 7: Ortofoto_2011 (Ortofoto 2011)
> 8: Ortofoto_2012 (Ortofoto 2012)
> 9: Ortofoto_2020 (Ortofoto 2020)
> 10: Ortofoto_2020_CIR (Ortofoto 2020 CIR)
> 11: SLOBODNI_POSLOVNI_PROSTORI (SLOBODNI_POSLOVNI_PROSTORI)
> 12: ULAZI (Adrese)
> 13: ULICE (Ulice)
> 14: WIFI_PODRUCJA_POKRIVENOSTI (WiFi_PODRUCJA_POKRIVENOSTI)
> 15: WIFI_SSID_RIJEKA-FREE_ACCESS (WiFi SSID RIJEKA-FREE ACCESS)
> 16: WIFI_SSID_WIFI4EU (WiFi SSID WIFI4EU)
> 17: ZGRADE (Zgrade

`CKAN:Ortofoto_2012` seems not included in this list. Maybe `Ortofoto_2012` is the layer you intended to use.

File: sources/europe/hr/gis-rijeka-2000.geojson [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    "properties": {
+        "category": "historicphoto",
+        "id": "gis-rijeka-2000",
+        "name": "gis-rijeka: Rijeka 2000 aerial imagery",
+        "icon": "https://osmlab.github.io/editor-layer-index/sources/europe/hr/rijeka.png",
+        "start_date": "2000",
+        "end_date": "2000",
+        "url": "http://gis01.rijeka.hr/geoserver/CKAN/wms?service=WMS&version=1.1.0&request=GetMap&layers=Ortofoto_2000&styles=&bbox={bbox}&width={width}&height={height}&srs={proj}&format=image/png",
+        "permission_osm": "explicit",
+        "license_url": "http://opendefinition.org/licenses/cc-by/",
```
Comment (rbuffat) [inline sources/europe/hr/gis-rijeka-2000.geojson]: The `license_url` should point to an URL that describes under what license the data can be used, not the license itself.

**Comment by Janjko** [comment]
Seems good now, I can still fix the boundary geometries to be custom for each wms layer.

**Comment by Janjko** [comment]
Two more errors. One is the missing icon url that is already in the PR. I can merge the icon beforehand, but that seems superfluous. The other is City of Rijeka official website that decided to crash this morning and ruin my PR.

Here is the proof that the privacy policy web usually exists:
https://web.archive.org/web/20211021030430/https://www.rijeka.hr/teme-za-gradane/aktivno-gradanstvo/zastita-osobnih-podataka/

Edit: And it's live now: https://www.rijeka.hr/teme-za-gradane/aktivno-gradanstvo/zastita-osobnih-podataka/

**Comment by cicku** [comment]
Are those from 20 years ago really useful for OSM? I doubt it.

**Comment by Janjko** [comment]
> Are those from 20 years ago really useful for OSM? I doubt it.

If an underground cable is buried under a road, maybe there are traces of that on the ground 20 years ago. Maybe the imagery 20 years ago had less foliage so the path in a forest had better visibility. Maybe the angle of the photo uncovers one side of the building. There are a lot of reasons to have as much imagery as possible.

I didn't realize it burdened anyone to have a lot of imagery. Maybe the boundaries get too heavy? That's why I wanted to put one boundary for all these imageries, but there is no way to do that right now.

If you still think it's better to remove some, I can do that. Better than no imagery :) I'll analyze them a bit to see if some are not useful at all.

**Comment by cicku** [comment]
The general practice is less than 3. 2012 and 2020 can definitely get in without anything elder. If there is a new release for 2022, then 2012 should be dropped and later becoming 2020 and 2022. All other dated imageries should not be carried here. There are many commercial imageries available for OSM, why do you think people will take a look back to 2000 just for invisible objects?

_The exception is that when a territory is covered by a composite of different years, of course everything has to be included regardless of quantity._

**Comment by Janjko** [comment]
It's been 5 months, can we merge these layers? The only check that is failing is the icon URL not being there, because it's in this pull request. I cut the number of imageries to 2. I see no issues any more.

**Comment by grischard** [comment]
Sorry for the long wait.

**Pull request merged**